spec/features/create_and_deliver_digests_spec.rb

```diff
@@ -18,17 +18,17 @@ def first_expected_daily_email_body(subscription_one, subscription_two)
 
       [Title one](http://www.dev.gov.uk/base-path)
 
-      Change note one: Description one
+      Description one
 
-      Updated at 10:00 am on 1 January 2017
+      10:00am, 1 January 2017: Change note one
 
       ---
 
       [Title two](http://www.dev.gov.uk/base-path)
 
-      Change note two: Description two
+      Description two
 
-      Updated at 09:00 am on 1 January 2017
+       9:00am, 1 January 2017: Change note two
```

Comment:
I suspect it doesn't matter when the email gets rendered, but should we strip the date output to avoid this leading space?

Reply:
Stupid Ruby. Why would it replace a digit with a space?? If it doesn't render in the email then I guess it doesn't matter.

Reply:
I'll leave this up to Keith. I'm not bothered either way.